### ISSUE TITLE
work with php 7.3 under docker

### ISSUE THREAD

**Issue by zeetabit** [gkralik/php7-sapnwrfc#104]
Hello!

When I run `make test` I get a lot of failed tests. Is it ok or needs something else?
I'm using docker container based on the debian with php 7.3, copied sources of SAP NetWeaver RFC SDK 7.50 and copied your repository.

Let's see the `make test` output:

```
root@b0c0cc04abdd:/php7-sapnwrfc# make test
/bin/bash /php7-sapnwrfc/libtool --mode=install cp ./sapnwrfc.la /php7-sapnwrfc/modules
cp ./.libs/sapnwrfc.so /php7-sapnwrfc/modules/sapnwrfc.so
cp ./.libs/sapnwrfc.lai /php7-sapnwrfc/modules/sapnwrfc.la
PATH="$PATH:/sbin" ldconfig -n /php7-sapnwrfc/modules
----------------------------------------------------------------------
Libraries have been installed in:
   /php7-sapnwrfc/modules

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the `-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the `LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the `LD_RUN_PATH' environment variable
     during linking
   - use the `-Wl,--rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to `/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------

Build complete.
Don't forget to run 'make test'.


=====================================================================
PHP         : /usr/local/bin/php 
PHP_SAPI    : cli
PHP_VERSION : 7.3.26
ZEND_VERSION: 3.3.26
PHP_OS      : Linux - Linux b0c0cc04abdd 4.19.121-linuxkit #1 SMP Tue Dec 1 17:50:32 UTC 2020 x86_64
INI actual  : /php7-sapnwrfc/tmp-php.ini
More .INIs  :   
---------------------------------------------------------------------
PHP         : /usr/local/bin/phpdbg 
PHP_SAPI    : phpdbg
PHP_VERSION : 7.3.26
ZEND_VERSION: 3.3.26
PHP_OS      : Linux - Linux b0c0cc04abdd 4.19.121-linuxkit #1 SMP Tue Dec 1 17:50:32 UTC 2020 x86_64
INI actual  : /php7-sapnwrfc/tmp-php.ini
More .INIs  : 
---------------------------------------------------------------------
CWD         : /php7-sapnwrfc
Extra dirs  : 
VALGRIND    : Not used
=====================================================================
TIME START 2021-02-12 17:35:17
=====================================================================
PASS SAPNWRFC\Exception extends \RuntimeException [tests/001_exception_extends_runtime_exception.phpt] 
PASS SAPNWRFC\Exception has private "errorInfo" property [tests/002_exception_has_private_error_info.phpt] 
PASS SAPNWRFC\Exception has getter for "errorInfo" [tests/003_exception_has_getter_for_error_info.phpt] 
PASS SAPNWRFC\ConnectionException extends \SAPNWRFC\Exception [tests/004_connection_exception_extends_exception.phpt] 
PASS SAPNWRFC\FunctionCallException extends \SAPNWRFC\Exception [tests/005_functioncall_exception_extends_exception.phpt] 
PASS Exception "errorInfo" is array with at least "key", "code" and "message". [tests/006_exception_error_info_is_array.phpt] 
PASS SAPNWRFC\Connection has constants for trace levels. [tests/007_connection_has_constants_for_trace_levels.phpt] 
PASS version() returns version string x.y.z [tests/008_version_returns_version_info.phpt] 
PASS rfcVersion() returns SDK version string x.y.z [tests/009_rfc_version_returns_sdk_version_info.phpt] 
PASS Connection without parameters throws ConnectionException. [tests/010_connection_without_parameters_throws_connection_exception.phpt] 
PASS Connection with a non-array parameter throws ConnectionException. [tests/011_connection_with_non_array_parameter_throws_connection_exception.phpt] 
PASS Connection with empty array throws ConnectionException [tests/012_connection_throws_connection_exception_for_empty_array.phpt] 
PASS Connection throws ConnectionException with errorInfo set if connection fails. [tests/013_connection_throws_exception_and_has_error_info_if_connection_fails.phpt] 
FAIL Connection object is returned if connection is successful. [tests/014_connection_returns_object_if_ok.phpt] 
FAIL close() closes the connection and returns true if the connection was open. [tests/015_close_closes_connection_and_returns_true.phpt] 
FAIL close() returns false if connection was already closed. [tests/016_close_returns_false_if_connection_closed.phpt] 
FAIL close() does not accept parameters. [tests/017_close_does_not_accept_parameters.phpt] 
FAIL getAttributes() returns an array of connection attributes. [tests/018_connection_attributes.phpt] 
FAIL getAttributes() throws ConnectionException if connection already closed. [tests/019_attributes_throws_exception_if_connection_closed.phpt] 
FAIL getAttributes() does not take arguments. [tests/020_attributes_does_not_take_arguments.phpt] 
FAIL ping() returns true if connection is open. [tests/021_ping_returns_true.phpt] 
FAIL ping() throws ConnectionException if connection is closed. [tests/022_ping_throws_exception_if_connection_closed.phpt] 
FAIL ping() does not accept arguments. [tests/023_ping_does_not_accept_arguments.phpt] 
PASS setIniPath() returns true if path is correct and INI file is loaded. [tests/024_setinipath_returns_true_if_ok.phpt] 
PASS setIniPath() does only accept a string parameter. [tests/025_setinipath_only_accepts_string_parameter.phpt] 
PASS reloadIniFile() returns true if reload OK. [tests/026_reloadinifile_returns_true_if_ok.phpt] 
PASS reloadIniFile() does not accept parameters. [tests/027_reloadinifile_does_not_accept_parameters.phpt] 
FAIL getFunction() accepts only string parameter. [tests/028_get_function_throws_exception_if_not_passed_string.phpt] 
FAIL getFunction() throws FunctionCallException if function does not exist. [tests/029_get_function_throws_exception_if_function_does_not_exist.phpt] 
FAIL getFunction() throws FunctionCallException if connection already closed. [tests/030_get_function_throws_exception_if_connection_closed.phpt] 
FAIL getFunction() returns a RemoteFunction object if function exists. [tests/031_get_function_returns_function_object.phpt] 
FAIL RemoteFunction::getName() returns the function's name. [tests/032_function_object_has_name_property.phpt] 
FAIL Boolean parameters can be passed in connection options (eg trace => false) [tests/033_issue_13_can_pass_booleans_in_connection_params.phpt] 
FAIL Integer parameters can be passed in connection options (eg trace => 0) [tests/034_issue_13_can_pass_integers_in_connection_params.phpt] 
FAIL invoke() throws FunctionCallException if connection closed. [tests/035_invoke_throws_exception_if_connection_closed.phpt] 
FAIL invoke() accepts only array parameter. [tests/036_invoke_throws_exception_if_parameter_is_not_array.phpt] 
FAIL invoke() throws FunctionCallException if parameter key is no string. [tests/038_invoke_requires_all_parameter_keys_to_be_strings.phpt] 
FAIL invoke() throws FunctionCallException given an invalid RFC parameter. [tests/039_invoke_throws_exception_if_given_invalid_rfc_parameter.phpt] 
FAIL invoke() can call a RFC function. [tests/040_invoke_can_call_function.phpt] 
FAIL invoke() returns correct result for STFC_STRUCTURE (struture + tables). [tests/041_invoke_returns_result.phpt] 
FAIL invoke() can handle CHANGING parameter. [tests/042_invoke_can_handle_changing_parameter.phpt] 
FAIL invoke() throws FunctionCallException on ABAP error and sets "errorInfo" in exception. [tests/043_invoke_throws_exception_on_abab_error.phpt] 
FAIL invoke() does not return a value for an inactive parameter. [tests/044_invoke_does_not_return_a_value_for_an_inactive_parameter.phpt] 
FAIL A function handle can be reused (ie called more than once). [tests/046_function_handle_can_be_used_multiple_times.phpt] 
FAIL Throws FunctionCallException if not given string for RFC CHAR parameter. [tests/047_throws_exception_if_not_given_string_for_char.phpt] 
FAIL Throws FunctionCallException if not given string for RFC DATE parameter. [tests/048_throws_exception_if_not_given_string_for_date.phpt] 
FAIL Throws FunctionCallException if given wrong format for RFC DATE. [tests/049_throws_exception_if_given_wrong_date_format.phpt] 
FAIL Throws FunctionCallException if not given string for RFC TIME. [tests/050_throws_exception_if_not_given_string_for_time.phpt] 
FAIL Throws FunctionCallException if given wrong format for RFC TIME. [tests/051_throws_exception_if_given_wrong_format_for_time.phpt] 
FAIL Throws FunctionCallException if not given string for RFC BYTE parameter. [tests/052_throws_exception_if_not_given_string_for_byte.phpt] 
FAIL Throws FunctionCallException if RFC BYTE parameter too long. [tests/053_throws_exception_if_byte_parameter_too_long.phpt] 
FAIL Throws FunctionCallException if not given array for RFC TABLE. [tests/054_throws_exception_if_not_given_array_for_table.phpt] 
FAIL Throws FunctionCallException if not given array for RFC STRUCTURE. [tests/055_throws_exception_if_not_given_array_for_structure.phpt] 
SKIP Throws FunctionCallException if not given string for RFC NUM. [tests/056_throws_exception_if_not_given_string_for_num.phpt] reason: No suitable RFC to test.
SKIP Throws FunctionCallException if RFC NUM parameter too long. [tests/057_throws_exception_if_num_parameter_too_long.phpt] reason: No suitable RFC to test.
FAIL Throws FunctionCallException if given wrong type for RFC FLOAT parameter. [tests/058_throws_exception_if_not_given_int_or_double_for_float.phpt] 
FAIL Throws FunctionCallException if not given integer for RFC INT parameter. [tests/059_throws_exception_if_not_given_int_for_int.phpt] 
FAIL Throws FunctionCallException if not given integer for RFC INT1 parameter. [tests/060_throws_exception_if_not_given_int_for_int1.phpt] 
FAIL Throws FunctionCallException if not given integer for RFC INT2 parameter. [tests/061_throws_exception_if_not_given_int_for_int2.phpt] 
FAIL Throws FunctionCallException if integer is out of range (0 - 255) for RFC INT1. [tests/062_throws_exception_if_int1_parameter_out_of_range.phpt] 
FAIL Throws FunctionCallException if integer is out of range (-32767 - 32767) for RFC INT2 [tests/063_throws_exception_if_int2_parameter_out_of_range.phpt] 
FAIL Throws FunctionCallException if not given string for RFC STRING parameter. [tests/064_throws_exception_if_not_given_string_for_string.phpt] 
FAIL Throws FunctionCallException if not given string for RFC XSTRING parameter. [tests/065_throws_exception_if_not_given_string_for_xstring.phpt] 
FAIL Throws FunctionCallException if given invalid format for RFC XSTRING parameter. [tests/066_throws_exception_if_given_wrong_xstring_format.phpt] 
FAIL Throws FunctionCallException if setting unkown field for RFC TABLE. [tests/067_throws_exception_if_setting_unknows_field_for_table.phpt] 
FAIL Throws FunctionCallException if setting unkown field for RFC STRUCTURE. [tests/068_throws_exception_if_setting_unknows_field_for_structure.phpt] 
FAIL setParameterActive() takes a string and a boolean argument. [tests/069_set_active_takes_string_and_bool.phpt] 
FAIL setParameterActive() can set parameter status. [tests/070_set_active_can_set_parameter_status.phpt] 
FAIL setParameterActive() throws FunctionCallException if connection is closed. [tests/071_set_active_throws_exception_if_connection_closed.phpt] 
FAIL isParameterActive() accepts only a string argument. [tests/072_is_active_takes_string.phpt] 
FAIL isParameterActive() throws FunctionCallException if connection is closed. [tests/073_is_active_throws_exception_if_connection_closed.phpt] 
FAIL isParameterActive() returns parameter status. [tests/074_is_active_returns_parameter_status.phpt] 
FAIL setTraceDir() sets trace directory. [tests/075_set_trace_dir_sets_trace_directory.phpt] 
PASS setTraceDir() does only accept a string parameter. [tests/076_set_trace_dir_accepts_only_string.phpt] 
PASS setTraceDir() throws ConnectionException if path does not exist. [tests/077_set_trace_dir_throws_exception_if_directory_does_not_exist.phpt] 
FAIL setTraceLevel() changes the trace level. [tests/078_set_trace_level_changes_the_trace_level.phpt] 
PASS setTraceLevel() does only accept a long parameter. [tests/079_set_trace_level_accepts_long.phpt] 
PASS setTraceLevel() throws ConnectionException if given an invalid trace level. [tests/080_set_trace_level_throws_exception_if_invalid_level_given.phpt] 
FAIL Allows int or double for FLOAT parameter. [tests/081_allows_int_for_float_parameter.phpt] 
FAIL Allow passing references as parameters. [tests/082_allow_passing_references_as_parameters.phpt] 
FAIL RemoteFunction::invoke() takes optional $options array [tests/083_remote_function_invoke_takes_optional_options_array.phpt] 
FAIL rtrim is disabled by default. [tests/084_rtrim_disabled_by_default.phpt] 
FAIL Returned CHAR value is right-trimmed when rtrim option enabled. [tests/085_returned_CHAR_is_not_right_trimmed_when_rtrim_option_disabled.phpt] 
FAIL Can call invoke() without parameters [tests/086_can_call_invoke_without_parameters.phpt] 
FAIL UTF-8 and UTF-16 strings are handled correctly. [tests/087_can_handle_utf8_and_utf16_strings.phpt] 
FAIL RemoteFunction::getName() returns the function's name. [tests/088_function_has_accessor_for_name.phpt] 
SKIP Connection constructor takes options hash as second parameter. [tests/cache/001_connection_constructor_accepts_options_hash.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
SKIP An existing Function can be called after the cache was cleared. [tests/cache/002_function_can_be_invoked_after_clearing_cache.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
SKIP Clearing the function desc cache works. [tests/cache/003_clear_function_desc_cache.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
SKIP Looking up a function with disabled function desc cache works [tests/cache/004_lookup_function_with_cache_disabled.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
PASS clearFunctionDescCache function exists. [tests/cache/005_clear_function_desc_cache_function_exists.phpt] 
SKIP decfloat: returns string [tests/decfloat/001_returns_string.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
SKIP decfloat: accepts string, int or double [tests/decfloat/002_accepts_string_int_double.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
PASS RemoteFunction::getFunctionDescription() exists [tests/function_description/001_method_exists.phpt] 
SKIP RemoteFunction::getFunctionDescription() returns empty array for functions without parameters. [tests/function_description/002_returns_empty_array_for_function_without_parameters.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
SKIP RemoteFunction::getFunctionDescription() returns function interface as array. [tests/function_description/003_returns_function_interface.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
SKIP RemoteFunction::getFunctionDescription() returns typedef for structures/tables. [tests/function_description/004_complex_types_contain_typedef.phpt] reason: NO ONLINE TESTS: create configuration `sapnwrfc.config.inc` file to enable.
=====================================================================
TIME END 2021-02-12 17:35:44

=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :    0
Exts tested     :   33
---------------------------------------------------------------------

Number of tests :   97                86
Tests skipped   :   11 ( 11.3%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :   63 ( 64.9%) ( 73.3%)
Expected fail   :    0 (  0.0%) (  0.0%)
Tests passed    :   23 ( 23.7%) ( 26.7%)
---------------------------------------------------------------------
Time taken      :   27 seconds
=====================================================================

=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
Connection object is returned if connection is successful. [tests/014_connection_returns_object_if_ok.phpt]
close() closes the connection and returns true if the connection was open. [tests/015_close_closes_connection_and_returns_true.phpt]
close() returns false if connection was already closed. [tests/016_close_returns_false_if_connection_closed.phpt]
close() does not accept parameters. [tests/017_close_does_not_accept_parameters.phpt]
getAttributes() returns an array of connection attributes. [tests/018_connection_attributes.phpt]
getAttributes() throws ConnectionException if connection already closed. [tests/019_attributes_throws_exception_if_connection_closed.phpt]
getAttributes() does not take arguments. [tests/020_attributes_does_not_take_arguments.phpt]
ping() returns true if connection is open. [tests/021_ping_returns_true.phpt]
ping() throws ConnectionException if connection is closed. [tests/022_ping_throws_exception_if_connection_closed.phpt]
ping() does not accept arguments. [tests/023_ping_does_not_accept_arguments.phpt]
getFunction() accepts only string parameter. [tests/028_get_function_throws_exception_if_not_passed_string.phpt]
getFunction() throws FunctionCallException if function does not exist. [tests/029_get_function_throws_exception_if_function_does_not_exist.phpt]
getFunction() throws FunctionCallException if connection already closed. [tests/030_get_function_throws_exception_if_connection_closed.phpt]
getFunction() returns a RemoteFunction object if function exists. [tests/031_get_function_returns_function_object.phpt]
RemoteFunction::getName() returns the function's name. [tests/032_function_object_has_name_property.phpt]
Boolean parameters can be passed in connection options (eg trace => false) [tests/033_issue_13_can_pass_booleans_in_connection_params.phpt]
Integer parameters can be passed in connection options (eg trace => 0) [tests/034_issue_13_can_pass_integers_in_connection_params.phpt]
invoke() throws FunctionCallException if connection closed. [tests/035_invoke_throws_exception_if_connection_closed.phpt]
invoke() accepts only array parameter. [tests/036_invoke_throws_exception_if_parameter_is_not_array.phpt]
invoke() throws FunctionCallException if parameter key is no string. [tests/038_invoke_requires_all_parameter_keys_to_be_strings.phpt]
invoke() throws FunctionCallException given an invalid RFC parameter. [tests/039_invoke_throws_exception_if_given_invalid_rfc_parameter.phpt]
invoke() can call a RFC function. [tests/040_invoke_can_call_function.phpt]
invoke() returns correct result for STFC_STRUCTURE (struture + tables). [tests/041_invoke_returns_result.phpt]
invoke() can handle CHANGING parameter. [tests/042_invoke_can_handle_changing_parameter.phpt]
invoke() throws FunctionCallException on ABAP error and sets "errorInfo" in exception. [tests/043_invoke_throws_exception_on_abab_error.phpt]
invoke() does not return a value for an inactive parameter. [tests/044_invoke_does_not_return_a_value_for_an_inactive_parameter.phpt]
A function handle can be reused (ie called more than once). [tests/046_function_handle_can_be_used_multiple_times.phpt]
Throws FunctionCallException if not given string for RFC CHAR parameter. [tests/047_throws_exception_if_not_given_string_for_char.phpt]
Throws FunctionCallException if not given string for RFC DATE parameter. [tests/048_throws_exception_if_not_given_string_for_date.phpt]
Throws FunctionCallException if given wrong format for RFC DATE. [tests/049_throws_exception_if_given_wrong_date_format.phpt]
Throws FunctionCallException if not given string for RFC TIME. [tests/050_throws_exception_if_not_given_string_for_time.phpt]
Throws FunctionCallException if given wrong format for RFC TIME. [tests/051_throws_exception_if_given_wrong_format_for_time.phpt]
Throws FunctionCallException if not given string for RFC BYTE parameter. [tests/052_throws_exception_if_not_given_string_for_byte.phpt]
Throws FunctionCallException if RFC BYTE parameter too long. [tests/053_throws_exception_if_byte_parameter_too_long.phpt]
Throws FunctionCallException if not given array for RFC TABLE. [tests/054_throws_exception_if_not_given_array_for_table.phpt]
Throws FunctionCallException if not given array for RFC STRUCTURE. [tests/055_throws_exception_if_not_given_array_for_structure.phpt]
Throws FunctionCallException if given wrong type for RFC FLOAT parameter. [tests/058_throws_exception_if_not_given_int_or_double_for_float.phpt]
Throws FunctionCallException if not given integer for RFC INT parameter. [tests/059_throws_exception_if_not_given_int_for_int.phpt]
Throws FunctionCallException if not given integer for RFC INT1 parameter. [tests/060_throws_exception_if_not_given_int_for_int1.phpt]
Throws FunctionCallException if not given integer for RFC INT2 parameter. [tests/061_throws_exception_if_not_given_int_for_int2.phpt]
Throws FunctionCallException if integer is out of range (0 - 255) for RFC INT1. [tests/062_throws_exception_if_int1_parameter_out_of_range.phpt]
Throws FunctionCallException if integer is out of range (-32767 - 32767) for RFC INT2 [tests/063_throws_exception_if_int2_parameter_out_of_range.phpt]
Throws FunctionCallException if not given string for RFC STRING parameter. [tests/064_throws_exception_if_not_given_string_for_string.phpt]
Throws FunctionCallException if not given string for RFC XSTRING parameter. [tests/065_throws_exception_if_not_given_string_for_xstring.phpt]
Throws FunctionCallException if given invalid format for RFC XSTRING parameter. [tests/066_throws_exception_if_given_wrong_xstring_format.phpt]
Throws FunctionCallException if setting unkown field for RFC TABLE. [tests/067_throws_exception_if_setting_unknows_field_for_table.phpt]
Throws FunctionCallException if setting unkown field for RFC STRUCTURE. [tests/068_throws_exception_if_setting_unknows_field_for_structure.phpt]
setParameterActive() takes a string and a boolean argument. [tests/069_set_active_takes_string_and_bool.phpt]
setParameterActive() can set parameter status. [tests/070_set_active_can_set_parameter_status.phpt]
setParameterActive() throws FunctionCallException if connection is closed. [tests/071_set_active_throws_exception_if_connection_closed.phpt]
isParameterActive() accepts only a string argument. [tests/072_is_active_takes_string.phpt]
isParameterActive() throws FunctionCallException if connection is closed. [tests/073_is_active_throws_exception_if_connection_closed.phpt]
isParameterActive() returns parameter status. [tests/074_is_active_returns_parameter_status.phpt]
setTraceDir() sets trace directory. [tests/075_set_trace_dir_sets_trace_directory.phpt]
setTraceLevel() changes the trace level. [tests/078_set_trace_level_changes_the_trace_level.phpt]
Allows int or double for FLOAT parameter. [tests/081_allows_int_for_float_parameter.phpt]
Allow passing references as parameters. [tests/082_allow_passing_references_as_parameters.phpt]
RemoteFunction::invoke() takes optional $options array [tests/083_remote_function_invoke_takes_optional_options_array.phpt]
rtrim is disabled by default. [tests/084_rtrim_disabled_by_default.phpt]
Returned CHAR value is right-trimmed when rtrim option enabled. [tests/085_returned_CHAR_is_not_right_trimmed_when_rtrim_option_disabled.phpt]
Can call invoke() without parameters [tests/086_can_call_invoke_without_parameters.phpt]
UTF-8 and UTF-16 strings are handled correctly. [tests/087_can_handle_utf8_and_utf16_strings.phpt]
RemoteFunction::getName() returns the function's name. [tests/088_function_has_accessor_for_name.phpt]
=====================================================================

You may have found a problem in PHP.
This report can be automatically sent to the PHP QA team at
http://qa.php.net/reports and http://news.php.net/php.qa.reports
This gives us a better understanding of PHP's behavior.
If you don't want to send the report immediately you can choose
option "s" to save it.	You can then email it to qa-reports@lists.php.net later.
Do you want to send this report now? [Yns]:n 
make: *** [Makefile:134: test] Error 1

```

**Comment by gkralik** [comment]
Check `tests/*.log` for details about what's going on.

**Comment by zeetabit** [comment]
I found that all of these tests are requires setup connection and all them should be skipped by file https://github.com/gkralik/php7-sapnwrfc/blob/main/tests/014_connection_returns_object_if_ok.phpt . Seems everything is ok on that, but they are launched every time.

The first logfile for ex.:
```
cat 014_connection_returns_object_if_ok.log 

---- EXPECTED OUTPUT
ok
---- ACTUAL OUTPUT
Warning: include(/usr/src/php7-sapnwrfc-1.4.0/tests/sapnwrfc.config.inc): failed to open stream: No such file or directory in /usr/src/php7-sapnwrfc-1.4.0/tests/014_connection_returns_object_if_ok.php on line 2

Warning: include(): Failed opening '/usr/src/php7-sapnwrfc-1.4.0/tests/sapnwrfc.config.inc' for inclusion (include_path='.:/usr/share/php7') in /usr/src/php7-sapnwrfc-1.4.0/tests/014_connection_returns_object_if_ok.php on line 2

Fatal error: Uncaught TypeError: Argument 1 passed to SAPNWRFC\Connection::__construct() must be of the type array, bool given in /usr/src/php7-sapnwrfc-1.4.0/tests/014_connection_returns_object_if_ok.php:4
Stack trace:
#0 /usr/src/php7-sapnwrfc-1.4.0/tests/014_connection_returns_object_if_ok.php(4): SAPNWRFC\Connection->__construct(false)
#1 {main}
  thrown in /usr/src/php7-sapnwrfc-1.4.0/tests/014_connection_returns_object_if_ok.php on line 4
---- FAILED
```
-------
Could you help with this? I use the debian based container. Is it ok wheb I have a dev_rfc.log file after run out php?

```
root@1122cb406abd:/usr/src/php7-sapnwrfc-1.4.0/tests# cat dev_rfc.log 

**** Log file opened at 2021-02-15 14:39:21.624909 UTC+00:00 (GMT), Encoding UTF-8
NW RFC Library: SDK variant, Release 750 Patch Level 7, Compiled on Sep 19 2020 00:13:35
 CPIC library: 753.2020.09.14 version 3, NI library: 40, Kernel Release: 753 Patch Level 711
 Current working directory: /usr/src/php7-sapnwrfc-1.4.0/tests, Program: php
 Hardware: AMD/Intel x86_64 with Linux x86_64, Operating_system: Linux 4.19.121-linuxkit
 Hostname: 1122cb406abd, IP address: 172.17.0.3, IPv6 address: 

2021-02-15 14:39:21.624929 [139790973494336] >> Error entry
	The UUID daemon (uuidd) is not active.
Please ask your system administrator to activate uuidd according to SAP note 1391070.
```

**Comment by gkralik** [comment]
Thanks for the logs. I'll check why the tests are not skipped...

The `dev_rfc.log` file is expected. Ignore it or install `uuidd` as mentioned in the file.

**Comment by gkralik** [comment]
Obviously, I messed up the skipif includes (again). Should be fixed with 5deb513.
